projects/imagemagick/encoder_fuzzer.cc

```diff
@@ -0,0 +1,24 @@
+#include <cstdint>
+
+#include <Magick++/Blob.h>
+#include <Magick++/Image.h>
+
+#define FUZZ_ENCODER_STRING_LITERAL(name) #name
+#define FUZZ_ENCODER FUZZ_ENCODER_STRING_LITERAL(FUZZ_IMAGEMAGICK_ENCODER)
```

Comment:
Is this a correct macro?

Reply:
I'm pretty sure it does the right thing.